Makefile

```diff
@@ -785,10 +785,11 @@ test-postgres: test-postgres-docker
 
 test-migrations: test-postgres-docker
 	echo "--- test migrations"
+	set -euxo pipefail
```

Comment:
Is this safe here? We won't leak any CI secrets accidentally?

Reply:
I'll remove the `-x` just to be sure.